pkg/scheduler/scheduler.go

```diff
@@ -272,12 +272,14 @@ func (sched *Scheduler) Config() *factory.Config {
 func (sched *Scheduler) recordSchedulingFailure(pod *v1.Pod, err error, reason string, message string) {
 	sched.config.Error(pod, err)
 	sched.config.Recorder.Eventf(pod, nil, v1.EventTypeWarning, "FailedScheduling", "Scheduling", message)
-	sched.config.PodConditionUpdater.Update(pod, &v1.PodCondition{
+	if err := sched.config.PodConditionUpdater.Update(pod, &v1.PodCondition{
```

Comment:
LGTM. I'm curious how often does it happen in your env?

Reply:
sometimes because of lack of the status update permission (customized scheduler deployment)